js_modules/dagster-ui/packages/ui-core/src/runs/RunsFeedRoot.tsx

```diff
@@ -153,23 +158,52 @@ export const RunsFeedRoot = () => {
   const totalHeight = rowVirtualizer.getTotalSize();
   const items = rowVirtualizer.getVirtualItems();
 
+  const [hideSubRuns, setHideSubRuns] = useState(false);
+
   function actionBar() {
+    const selectedRuns = entries.filter(
+      (e): e is RunsFeedTableEntryFragment_Run => checkedIds.has(e.id) && e.__typename === 'Run',
+    );
+    const backfillsExcluded = entries.length - selectedRuns.length;
     return (
       <Box flex={{direction: 'column', gap: 8}}>
         <Box
           flex={{justifyContent: 'space-between'}}
           style={{width: '100%'}}
           padding={{left: 24, right: 12}}
         >
-          {button}
+          <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}}>
+            {button}
+            <Checkbox
+              label={<span>Hide sub-runs</span>}
```

Comment:
What is a sub-run?

Reply:
we may need to come up with a better name for it. It's intended that a "subrun" is any run that is part of a backfill. It originated when we were trying to eliminate the use of "backfill" entirely, but needed a way to still refer to runs that were launched as part of a larger coordinated process

Reply:
I think "child runs" would be better?

Reply:
yeah maybe. I'd like to get some other opinions on it, I put it as a discussion topic for our sync tomorrow